src/components/UNSTABLE_Pagination/index.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @file Unstable pagination entry point.
+ * @copyright IBM Security 2019
+ */
+
+import UNSTABLE_Pagination from './Pagination';
+import PageSelector from './PageSelector';
+
+export { PageSelector, UNSTABLE_Pagination };
```

Comment:
Would you mind exporting the `UNSTABLE_Pagination` as the default export and the `PageSelector` as named? I believe this aligns with other components.